source/core/parameters.cc

```diff
@@ -239,6 +239,33 @@ namespace Parameters
     prm.leave_subsection();
   }
 
+  void
+  PowerLawParameters::declare_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("power-law");
+    {
+      prm.declare_entry("K", "1.0", Patterns::Double(), "Viscosity at rest");
```

Comment:
```suggestion
      prm.declare_entry("K", "1.0", Patterns::Double(), "Viscosity at rest");
```
Is K really the viscosity at rest? Generally we call it the fluid consistency index : https://en.wikipedia.org/wiki/Power-law_fluid  . N is generally the flow behavior index.

Reply:
You are right, actually K is the viscosity when n = 1. I will change both!